recipe/meta.yaml

```diff
@@ -192,13 +203,7 @@ outputs:
       run:
         - openblas   # [blas_impl == "openblas"]
         - mkl-devel  # [blas_impl == "mkl"]
-        - {{ pin_subpackage("liblapack", exact=True) }}      # [blas_impl != 'blis']
-        - {{ pin_subpackage("liblapacke", exact=True) }}     # [blas_impl != 'blis']
-        - liblapack  {{ version }} *netlib                   # [blas_impl == 'blis']
-        - liblapacke {{ version }} *netlib                   # [blas_impl == 'blis']
-        - {{ pin_subpackage("libcblas", exact=True) }}
-        - {{ pin_subpackage("libblas", exact=True) }}
-        - {{ pin_subpackage("blas-devel", exact=True) }}
+        - {{ pin_subpackage("blas", exact=True) }}
```

Comment:
@isuruf, not sure I understood your intention from #61 here correctly, but `blas-devel` depending on itself sounds fragile and circular. My understanding would be that it builds on `blas` and adds the "development headers" (or whatever is the equivalent)...?

Reply:
Yes that's a mistake and should be removed. `blas` should depend on `blas-devel`.

Reply:
I thought it was the other way around? Kinda like `blas-devel` > `blas` (in terms of dependencies)

At least that's how I've understood the `-dev[el]` packages in various distros, for example.